model/distro/distro.go

```diff
@@ -337,22 +387,37 @@ func (d *Distro) GetResolvedPlannerSettings(config evergreen.SchedulerConfig) (P
 		TargetTime:             ps.TargetTime,
 		AcceptableHostIdleTime: ps.AcceptableHostIdleTime,
 		GroupVersions:          ps.GroupVersions,
-		PatchZipperFactor:      ps.PatchZipperFactor,
 		TaskOrdering:           ps.TaskOrdering,
+		PatchZipperFactor:      ps.PatchZipperFactor,
+		TimeInQueueFactor:      ps.TimeInQueueFactor,
+		ExpectedRuntimeFactor:  ps.ExpectedRuntimeFactor,
+		maxDurationPerHost:     evergreen.MaxDurationPerDistroHost,
+	}
+
+	catcher := grip.NewBasicCatcher()
+
+	if d.ContainerPool != "" {
+		pool := s.ContainerPools.GetContainerPool(d.ContainerPool)
```

Comment:
Is this just to validate the pool exists? You don't use this variable later. Is there some case where it might not exist?